README.md

```diff
@@ -22,13 +26,13 @@ this is in no way an endorsement of any of these programs, organizations, or the
 * [HealthEquity Intern - Security Operations](https://careers-healthequity.icims.com/jobs/5218/intern---security-operations/job?mobile=false&width=995&height=500&bga=true&needsRedirect=false&jan1offset=-300&jun1offset=-240) - Remote
 * [Mandiant/Google](https://careers.google.com/jobs/results/110830639655068358-security-consultant-intern-summer-2023/) NYC, Washington DC, San Francisco, or Reston, VA
 * [Microsoft](https://microsoft.talentify.io/job/research-intern-cybersecurity-machine-learning-redmond-washington-microsoft-1496779) - Research Intern - Cybersecurity Machine Learning, Redmon, WA
-* [NFP Cyber Security Intern](https://www.linkedin.com/jobs/view/3443446856/) - Austin, TX Hybrid 
+* [NFP Cyber Security Intern](https://www.linkedin.com/jobs/view/3443446856/) - Austin, TX Hybrid
 
-## Tech internships but not cybersecurity specific
+### Tech internships but not cybersecurity specific
```

Comment:
please keep this at the level of "internships" since it's decidedly NOT a cybersec internship, just a tech co internship that caught my eye.